python/mxnet/libinfo.py

```diff
@@ -97,9 +97,16 @@ def find_include_path():
 
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     incl_path = os.path.join(curr_path, '../../include/')
-    if not os.path.isdir(incl_path):
-        raise RuntimeError('Cannot find the MXNet include path.\n')
-    return incl_path
+    if os.path.isdir(incl_path):
+        # include path if build from source
+        return incl_path
+    else:
+        incl_path = os.path.join(curr_path, '../include/')
```

Comment:
does pip package really have a include directory?

Reply:
Yes, thanks to @szha who added it several days ago.

You may find it at: <mxnet_package_location>/lib/python3.5/site-packages/mxnet/include